beast/tools/remove_filters.py

```diff
@@ -181,18 +181,18 @@ def remove_filters_from_files(
         nlamb = np.delete(g0.lamb, rindxs, 0)
         nfilters = np.delete(filters, rindxs, 0)
         for rcol in rgridcols:
-            g0.grid.delCol(rcol)
+            g0.grid.remove_column(rcol)
 
         print("orig filters: {}".format(" ".join(filters)))
         print(" new filters: {}".format(" ".join(nfilters)))
 
         # save the modified grid
-        g = SpectralGrid(np.array(nlamb), seds=nseds, grid=g0.grid, backend="memory")
-        g.grid.header["filters"] = " ".join(nfilters)
+        g = SEDGrid(np.array(nlamb), seds=nseds, grid=g0.grid, backend="memory")
```

Comment:
We might want to keep using the 'SpectralGrid' here?

Reply:
Can you say more about why we would use `SpectralGrid` instead?  My thinking was that if we are removing filters, then this is an `SEDGrid` as it isn't about spectra at this point.

Reply:
I think you are right. My brain used an automated mapping between the old and new codes: 'FileSEDGrid' to 'SEDGrid' and 'SpectralGrid' to 'SpectralGrid'.

Reply:
Ok.  Thanks for answering.